examples/basic-host/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="color-scheme" content="light dark">
```

Comment:
I made this change to all the examples so that they can properly have a transparent background. Without it, iframe sets a default black/white background